hw/ip/clkmgr/data/clkmgr_testplan.hjson

```diff
@@ -153,7 +153,14 @@
       milestone: V2
       tests: []
     }
+    {
+      name: stress
+      desc: '''Create standard stress tests.'''
```

Comment:
nit: perhaps add a TODO to add what sequences we use in stress. It's not clear right now, but we should know after we have all the sub-sequences.

Reply:
Done.